presto-product-tests/conf/presto/etc/jvm.config

```diff
@@ -15,6 +15,5 @@
 -XX:ReservedCodeCacheSize=150M
 -DHADOOP_USER_NAME=hive
 -Duser.timezone=Asia/Kathmandu
--Xdebug 
```

Comment:
Commit message
* sentence case
* what about "Do not use JVM debug options when running product tests"

also, you may want to remove `-Xdebug` in separate commit labelling eg "Remove no-op JVM flag from product tests execution"

Reply:
How about *"Do not enable JVM debugger when running product tests"*

Reply:
@electrum note that https://docs.oracle.com/javase/8/docs/technotes/tools/unix/java.html lists `-Xdebug` as "Does nothing. Provided for backward compatibility.", so removal of `-Xrunjdwp:transport:..` is the most important change here

Reply:
done